docs/mast/mast.rst

```diff
@@ -494,11 +492,11 @@ The returned fields vary by catalog, find the field documentation for specific c
 Some catalogs have a maximum number of results they will return.
 If a query results in this maximum number of results a warning will be displayed to alert the user that they might be getting a subset of the true result set.
 
-.. doctest-remote-data::
+.. doctest-skip::
```

Comment:
why switch to skip?

Reply:
Because even though the output includes the max results warning the testing infrastructure considers it an unexpected exception. And obviously I don't want to change the example because the point is to show the user the max results warning.

Reply:
`pytest-doctestplus` provides a directive for these kinds of situations, see https://github.com/astropy/pytest-doctestplus#showing-warnings